webui/react/src/pages/F_ExpList/glide-table/GlideTable.tsx

```diff
@@ -687,13 +727,31 @@ export const GlideTable: React.FC<GlideTableProps> = ({
             break;
         }
         switch (currentColumn.type) {
-          case V1ColumnType.NUMBER:
-            columnDefs[currentColumn.column] = defaultNumberColumn(
-              currentColumn,
-              columnWidths[currentColumn.column],
-              dataPath,
-            );
+          case V1ColumnType.NUMBER: {
+            const heatmap = projectHeatmap.find((h) => h.metricsName === currentColumn.column);
+            if (
+              (heatmapApplied === 'all' || heatmapApplied.includes(currentColumn.column)) &&
+              heatmap
```

Comment:
nit: perhaps check for the heatmap before checking if it's in the list?
```suggestion
              heatmap &&
              (heatmapApplied === 'all' ||
              heatmapApplied.includes(currentColumn.column))
```